hbase-server/src/main/java/org/apache/hadoop/hbase/mob/DefaultMobStoreCompactor.java

```diff
@@ -311,13 +311,9 @@ protected boolean performCompaction(FileDetails fd, InternalScanner scanner, Cel
         // logging at DEBUG level
         if (LOG.isDebugEnabled()) {
           if ((now - lastMillis) >= COMPACTION_PROGRESS_LOG_INTERVAL) {
-            LOG.debug("Compaction progress: "
-                + compactionName
-                + " "
-                + progress
-                + String.format(", rate=%.2f kB/sec", (bytesWrittenProgressForLog / 1024.0)
-                    / ((now - lastMillis) / 1000.0)) + ", throughputController is "
-                + throughputController);
+            double rate = (bytesWrittenProgressForLog / 1024.0) / ((now - lastMillis) / 1000.0);
```

Comment:
What abt the String formatting on the Double value which we were doing.  Missing that in log now.  Below one more place too.

Reply:
Emm... maybe I need to format the rate first before put it as a arg in LOG.debug, thanks .